### PR TITLE
C-1725 Fix sign-up with Instagram

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
For some reason we are only getting IG id and handle back from the auth response (so no profile pic, no full name, etc). Not sure if this is a change to the IG API that we have to account for in Identity, or maybe a regression in the Identity code or something else. But this fixes the silent failure on the UI side for now. The UX experience is still not great though because basically no info gets filled out from this.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/common/src/services/oauth/formatSocialProfile.ts [hunk 1]
```diff
@@ -109,10 +109,9 @@ export const formatInstagramProfile = async (
     instagramProfile.full_name.length > MAX_DISPLAY_NAME_LENGTH
   ) {
     requiresUserReview = true
-    instagramProfile.full_name = instagramProfile.full_name!.slice(
-      0,
-      MAX_DISPLAY_NAME_LENGTH
-    )
+    if (instagramProfile.full_name) {
+      instagramProfile.full_name.slice(0, MAX_DISPLAY_NAME_LENGTH)
```
Comment (dylanjeffers) [inline packages/common/src/services/oauth/formatSocialProfile.ts:113]: Slice doesn't change orignal array right?

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
Nice catch, small not for setting sliced value

File: packages/common/src/services/oauth/formatSocialProfile.ts (hunk 1 above)
Comment (nicoback2) [reply packages/common/src/services/oauth/formatSocialProfile.ts:113]: oh yeah lol

Comment (nicoback2) [reply packages/common/src/services/oauth/formatSocialProfile.ts]: i forgot to put the assignment back when i moved `instagramProfile.full_name` it to the if statement 💀 thx for catching

**Review by raymondjacobson — APPROVED** [review]
lgtm. we _should_ have this going forward with insta fixed

**Comment by nicoback2** [comment]
Ugh IG removed the other fields https://stackoverflow.com/questions/50086945/how-to-get-instagram-profile-picture-via-api https://developers.facebook.com/docs/instagram-basic-display-api/reference/user#fields

**Review by dylanjeffers — APPROVED** [review]
nicee

**Pull request merged**